plugins/shopify/src/plugin.tsx

```diff
@@ -85,10 +85,12 @@ Builder.register('plugin', {
       await Promise.all(promises);
 
       // import and register webhooks
+      appState.globalState.showGlobalBlockingLoadingIndicator = true;
       await importProducts('shopify-product');
       await createWebhooks('product', 'shopify-product');
       await importCollections('shopify-collection');
       await createWebhooks('collection', 'shopify-collection');
+      appState.globalState.showGlobalBlockingLoadingIndicator = false;
```

Comment:
might be good to handle errors here even with just a modal saying contact our support, or else there could be cases of infinite spinners